README.md

```diff
@@ -0,0 +1 @@
+THIS IS REPO WITH COOL ALGOS TO PRACTIVE FOR INTERVIEW PROCESS ( JUST KIDDING )HAHA
```

Comment:
Please use proper and professional language..
also use marker readme syntax for better indentation..

Reply:
Neerav I have already used marker syntax.. please check raw file..